skimage/measure/_colocalization.py

```diff
@@ -135,13 +135,14 @@ def manders_coloc_coeff(image0, image1_mask, mask=None):
     The implemented equation is:
 
     .. math::
-        r = \frac{\sum A_{i,coloc}}{\sum A_i}
+
+       r = \frac{\sum A_{i,coloc}}{\sum A_i}
 
     where
-        :math:`A_i` is the value of the :math:`i^{th}` pixel in `image0`
-        :math:`A_{i,coloc} = A_i` if :math:`Bmask_i > 0`
-        :math:`Bmask_i` is the value of the :math:`i^{th}` pixel in
-        `mask`
+
+    - :math:`A_i` is the value of the :math:`i^{th}` pixel in `image0`,
+    - :math:`A_{i,coloc} = A_i` if :math:`Bmask_i > 0`, and
+    - :math:`Bmask_i` is the value of the :math:`i^{th}` pixel in `mask`.
```

Comment:
```suggestion
    - :math:`A_{i,coloc} = A_i` if :math:`Bmask_i > 0`, :math:`Bmask_i` being the value of the :math:`i^{th}` pixel in `mask`.
```
I think that $Bmask_i$ shouldn't be an additional item on its own line, because it's not featured in the displayed equation (for a second, I was looking for $Bmask_i$ in the expression and couldn't find it, of course). It's just used to define $A_{i, coloc}$ -- which, by the way, raises a question: What is the value of $A_{i, coloc}$ if $Bmask_i \leq 0$? I'll take a screenshot so it's easier to see the problem:

![broken_eq](https://github.com/user-attachments/assets/50c4f675-011e-42ec-a7b0-b86f8cf3dfac)

Reply:
@stefanv do you give me permission to commit my own suggestion here? So I can approve this PR :smile:

Reply:
Go for it :)